content/learning-paths/servers-and-cloud-computing/cca-veraison/attestation-token.md

```diff
@@ -0,0 +1,148 @@
+{{% /notice %}}
+
+## Build the EVCLI Tool
+
+Now that you have downloaded the example CCA attestation token, the next step is to look inside the token and learn about the data that it contains. Because the token is a binary file, you will need to use a tool to parse the file and display its contents. The tool that you will use is a command-line tool called `evcli` (which is short for the EVidence Command Line Interface – remember that attestation tokens are used to convey evidence about realms and the platforms on which they are hosted).
+
+The `evcli` tool is part of the Veraison open-source project, which was covered in the previous section.
+
+Clone the source code using git as follows:
+
+```bash
+cd $HOME
+git clone https://github.com/veraison/evcli.git
+```
+Change the directory and build the tool:
+
+```bash
+cd evcli
+go build
+```
+
+The tool is quite small, so this should not take long. Once it has built, you can progress to the next step.
+
+## Inspect the CCA Example Attestation Token
+
+Now that you have built the `evcli` command-line tool, you can use it to inspect the contents of the example CCA attestation token that you downloaded earlier.
+
+Run the following command, taking care to substitute the correct path where you stored the CCA example token from the earlier step.
+
+```bash
+./evcli cca print --token $HOME/cca_example_token.cbor
+```
+
+The contents of the token are displayed as JSON. Check that the output matches the below. Some of the output has been removed for better readability.
+
+```output
+{
+  "cca-platform-token": {
+    "cca-platform-profile": "http://arm.com/CCA-SSD/1.0.0",
+    "cca-platform-challenge": "tZc8touqn8VVWHhrfsZ/aeQN9bpaqSHNDCf0BYegEeo=",
+    "cca-platform-implementation-id": "f0VMRgIBAQAAAAAAAAAAAAMAPgABAAAAUFgAAAAAAAA=",
+    "cca-platform-instance-id": "AQcGBQQDAgEADw4NDAsKCQgXFhUUExIREB8eHRwbGhkY",
+    "cca-platform-config": "z8/Pzw==",
+    "cca-platform-lifecycle": 12291,
+    "cca-platform-sw-components": [
+      {
+        "measurement-type": "RSE_BL1_2",
+        "measurement-value": "micfKpFrC27mzsskJvCzIG7wdFeL5V2byU9vP+Orhqo=",
+        "signer-id": "U3h5YwdTXfPsjYsVouLcVkFBnD0wYM/jIjjA+pc/eqM=",
+        "measurement-description": "sha-256"
+      },
+      (...)
+      {
+        "measurement-type": "SOC_FW_CONFIG",
+        "measurement-value": "5sIejSYP5xiC3r2zOdJAKiynZIUpvCMD9IZJvOA4ABc=",
+        "signer-id": "U3h5YwdTXfPsjYsVouLcVkFBnD0wYM/jIjjA+pc/eqM=",
+        "measurement-description": "sha-256"
+      }
+    ],
+    "cca-platform-service-indicator": "https://veraison.example/.well-known/veraison/verification",
+    "cca-platform-hash-algo-id": "sha-256"
+  },
+  "cca-realm-delegated-token": {
```

Comment:
Insert `   "cca-realm-profile": "tag:arm.com,2023:realm#1.0.0",` as the first data item nested within `cca-realm-delegated-token`

Reply:
(If in any doubt here, you could wholesale replace this chunk of sample output with the actual output that you obtain from `evcli` with the latest example data)

Reply:
done